src/subsystems/php/builders/simple/default.nix

```diff
@@ -136,13 +163,19 @@
           # disable packagist, set path repositories
           mv composer.json composer.json.orig
 
-          cat <<EOF >> $out/repositories.json
+          cat <<EOF > $out/repositories.json
           ${repositoriesString}
           EOF
+          cat <<EOF > $out/dependencies.json
```

Comment:
Could use [passAsFile](https://nixos.org/manual/nix/stable/language/advanced-attributes.html#adv-attr-passAsFile) indead?

Reply:
Thanks! That's much nicer.